src/routes/examples/overlapping_layer_events/+page.svelte

```diff
@@ -82,7 +103,20 @@
         on:mousemove={(e) => {
           lastEvent[i] = e.detail.features?.[0];
         }}
-      />
+      >
+        <Popup {openOn} {openIfTopMost} let:features>
+          <div style:background={color} style:color="white">
+            <p>{features.length} features from {color} layer</p>
+            {#if color == 'red'}
```

Comment:
To more clearly show overlapping popups, it's useful to make the lower ones larger. This was the simplest way I could think to do that

Reply:
Yeah this is tricky since you need to

1. Override Maplibre's popup positioning (could be done with offset I guess but the arrows would be wrong).
2. Add some layout engine to move them aside when multiple popups are visible
3. Do this only after the Popup has rendered, since before the first render you don't know how large it is
4. Make sure the popups aren't moved so far that they are off the screen

For these reasons, and since multiple popups at once is not really a common case anyway, I think it might actually be better to make `openIfTopMost` default to `true`. Any thoughts on that?

Reply:
To clarify, I don't think making multiple popups line up is a useful feature necessarily. I added coloring and sizing to the popups in this example just to demonstrate the bug/behavior when `openIfTopMost` is false.

Stuart also raised the idea that a useful UX pattern could be for one popup to receive and handle features from all layers. The real use case here is my main work project (which is now using svelte-maplibre!!), https://acteng.github.io/atip/browse.html. There are a bunch of overlapping layers that can be enabled, and currently the popup covers the top-most one. It might be less confusing and more useful (for someone using this app to understand small linear/point infrastructure or count data in context of a larger census-polygon area) to show info for everything under the cursor.

I think `openIfTopMost` is an intuitive default, so yes, I'll enable that now!

Reply:
Edit: I changed the changeset type to major, since this seems like a breaking change to popup behavior